scripts/gen-traffic.js

```diff
@@ -0,0 +1,68 @@
+const axios = require('axios').default;
+const Bee = require('@ethersphere/bee-js').Bee;
+
+/**
+ * Lehmer random number generator with seed (minstd_rand in C++11)
+ * !!! Very fast but not well distributed pseudo-random function !!!
+ *
+ * @param seed Seed for the pseudo-random generator
+ */
+function lrng(seed) {
+  return () => ((2 ** 31 - 1) & (seed = Math.imul(48271, seed))) / 2 ** 31
+}
+
+/**
+ * Utility function for generating random Buffer
+ * !!! IT IS NOT CRYPTO SAFE !!!
+ * For that use `crypto.randomBytes()`
+ *
+ * @param length Number of bytes to generate
+ * @param seed Seed for the pseudo-random generator
+ */
+function randomByteArray(length, seed = 500) {
+  const rand = lrng(seed)
+  const buf = new Uint8Array(length)
+
+  for (let i = 0; i < length; ++i) {
+    buf[i] = (rand() * 0xff) << 0
+  }
+
+  return buf
+}
+
+async function trafficGen(host = 'http://localhost:1633', seed = 500, bytes = 1024 * 4 * 400) {
+  const randomBytes = randomByteArray(bytes, seed)
+  const bee = new Bee(host)
+  const ref = await bee.uploadData(randomBytes)
+  console.log(`Generated ${bytes} bytes traffic, the random data's root reference: ${ref}`)
+}
+
+/**
+ * Generate traffic on Bee node(s)
+ * 
+ * @param beeApiUrls Bee API URLs where the random generated data will be sent to.
+ */ 
+async function genTrafficOnOpenPorts(beeApiUrls) {
+  for(const url of beeApiUrls) {
+    console.log(`Generate Swarm Chunk traffic on ${url}...`)
+    await trafficGen(url, new Date().getTime())
+  }
+}
+
+function sleep(ms) {
+  return new Promise(resolve => setTimeout(resolve, ms));
+}
+
+async function genTrafficLoop(hosts) {
+  while(true) {
+    const sleepMs = 1000
+
+    genTrafficOnOpenPorts(hosts)
```

Comment:
I'd suggest to wait for the execution here because you probably don't want to spam the bee node with more and more uploads if for any reason the previous ones are not finished yet.

```suggestion
    await genTrafficOnOpenPorts(hosts)
```

Reply:
This `await` is actually needed or the bee node is not able to handle the parallel requests (at least on my slow computer). I've also added one more suggestion in a PR targeting this branch. Feel free to merge it or just close it and implement the `await` change here. https://github.com/ethersphere/bee-factory/pull/27

After that I am fine with this PR and we can merge it.